src/bindgen/parser.rs

```diff
@@ -893,4 +913,39 @@ impl Parse {
             }
         }
     }
+
+    fn load_builtin_macro(
+        &mut self,
+        binding_crate_name: &str,
+        crate_name: &str,
+        mod_cfg: &Option<Cfg>,
+        item: &syn::ItemMacro,
+    ) {
+        let name = match item.mac.path.segments.last() {
+            Some(ref n) => n.value().ident.to_string(),
+            None => return,
+        };
+
+        if name != "bitflags" {
+            return;
+        }
+
+        let tokens = item.mac.tts.to_string();
+        let bitflags = match bitflags::parse(&tokens) {
+            Ok(b) => b,
+            Err(e) => {
+                warn!("Failed to parse bitflags invocation: {:?}", e);
+                return;
+            }
+        };
+
+        let (struct_, impl_) = bitflags.expand();
+        self.load_syn_struct(crate_name, mod_cfg, &struct_);
+        self.load_syn_assoc_consts_from_impl(
```

Comment:
IIUC, we defer loading assoc-consts from impls until we have parsed all item definitions because they may be out of order.

This code should work fine, because we load the struct the line before, and the expanded impl will only reference that struct. Might be worth a comment though.